cpp/src/strings/search/find.cu

```diff
@@ -271,6 +321,15 @@ std::unique_ptr<column> rfind(strings_column_view const& strings,
   return detail::rfind(strings, target, start, stop, cudf::get_default_stream(), mr);
 }
 
+std::unique_ptr<column> find(strings_column_view const& input,
```

Comment:
Do we need an `rfind` variation for target-per-row?

Reply:
This was not requested. I don't want to add this feature without a reason.

Reply:
Can you comment on the original issue to indicate that this can be done (the machinery appears to be in place) but we're not including in this PR because it was not requested? Just want to communicate that so that we aren't skipping over something that is easy and might be expected by the requesters.

Reply:
Ehhh maybe not. I thought some more, and I think this can be skipped.

Reply:
> This was not requested. I don't want to add this feature without a reason.

Typically, I always advocate for better coverage of the library feature.